modules/customProtocols.js

```diff
@@ -0,0 +1,46 @@
+"use strict";
+
+const electron = require('electron');
+const protocol = electron.protocol;
+const path = require('path');
+const Settings = require('./settings');
+const log = require('./utils/logger').create('customProtocols');
+
+
+const protocols = {
+    mist: (req, cb) => {
+        const url = (req.url.indexOf('mist://interface') !== -1) 
```

Comment:
This was already there so I kept it. It looks like it's a way for people to browse to specific screens within Mist via a URL.